.github/workflows/cookiecutter_docs.yml

```
@@ -0,0 +1,31 @@
+name: Documentation
+on:
+  push:
+    branches: [main]
+    tags: ['*']
```

Just so that it builds on PRs and can also be triggered manually.
```suggestion
on:
  push:
    branches:
      - main
    tags:
      - '*'
  pull_request:
  workflow_dispatch:
```